front_end/src/components/indiv-hotel/IndivHotel.js

```diff
@@ -0,0 +1,302 @@
+import React, { Component } from "react";
+import {Map, InfoWindow, Marker, GoogleApiWrapper} from 'google-maps-react';
+import ReactStars from 'react-stars';
+import AnchorLink from 'react-anchor-link-smooth-scroll'
+
+//  sliding image testing purpose
+// All images are local, since I have server connection issues
+// all image path are hardcoded as the moment
+// pending for further development
+//
+import images1 from "/Users/Alexnader/Abode/front_end/src/images/hotelImages/paris-1.jpg";
```

Comment:
imports have to be made from the view of the file, not your pc.
the path from IndivHotel.js to the images:

""../../images/hotelImages/paris-1.jpg""